pkg/controller/dataimportcron-controller_test.go

```diff
@@ -690,7 +690,7 @@ func createDataImportCronReconciler(objects ...runtime.Object) *DataImportCronRe
 
 func createDataImportCronReconcilerWithoutConfig(objects ...runtime.Object) *DataImportCronReconciler {
 	crd := &extv1.CustomResourceDefinition{ObjectMeta: metav1.ObjectMeta{Name: "dataimportcrons.cdi.kubevirt.io"}}
-	objs := []runtime.Object{crd}
+	objs := []runtime.Object{crd, cc.MakeEmptyCDICR()}
```

Comment:
Maybe unit test that existing cronjob (carry over from upgrade etc) gets reconciled with placement values?

Reply:
added